README.md

```diff
@@ -1 +1,53 @@
-# typewriter
\ No newline at end of file
+# Typewriter
+
+[![Build Status](https://travis-ci.org/j-delaney/typewriter.svg?branch=travis)](https://travis-ci.org/j-delaney/typewriter)
+
+A library and command-line tool for displaying 2 columns of text.
+The original purpose of this library is to improve failed test output when comparing two multiline strings.
+Go from this:
+
+![](/readme_imgs/before.png?raw=true "Before")
+
+To this:
+![](/readme_imgs/after.png?raw=true "After")
+
+## Installing The CLI
+
+If you just want the `side_by_side` command-line tool you should run this command:
+```
+go get -u github.com/j-delaney/typewriter/cmd/side_by_side
+```
+
+This will put the `side_by_side` tool in your `~/go/bin` directory.
```

Comment:
Can you also add how to run `side_by_side`? Sorry I'm a noob with Go 😛

Reply:
Good thinking. My next commit expands more on how to use the CLI